homeassistant/components/homekit_controller/strings.json

```diff
@@ -71,6 +71,30 @@
     }
   },
   "entity": {
+    "event": {
+      "doorbell": {
+        "state_attributes": {
+          "event_type": {
+            "state": {
+              "double_press": "Double Press",
+              "long_press": "Long Press",
+              "single_press": "Single Press"
+            }
+          }
+        }
+      },
+      "button": {
```

Comment:
If they are the same, we can just use one and the same translation key and omit one?

Reply:
When I do finish supporting translations for the name, no I'll need different translation keys. Is there a syntax to refer to the other keys within the same strings.json?

Reply:
Yes, there is, check out `light/strings.json`, I know many of them are in there... :)